babel.config.js

```diff
@@ -3,59 +3,44 @@ module.exports = {
     [
       '@babel/preset-env',
       {
-        useBuiltIns: 'usage',
-        corejs: 3,
-        targets: { node: '8' },
+        targets: { node: 18 },
       },
     ],
     '@babel/preset-typescript',
   ],
   plugins: [
-    '@babel/plugin-proposal-optional-chaining',
-    '@babel/plugin-proposal-nullish-coalescing-operator',
-    '@babel/plugin-proposal-class-properties',
     [
       '@babel/plugin-transform-runtime',
       {
-        corejs: 3,
+        corejs: "3",
       },
     ],
   ],
   env: {
-    module: {
-      presets: [
-        [
-          '@babel/preset-env',
-          {
-            modules: false,
-            useBuiltIns: 'usage',
-            corejs: 3,
-            targets: { node: '8' },
-          },
-        ],
-        '@babel/preset-typescript',
-      ],
-    },
```

Comment:
this is the babel config for the node ESM build done by the (now deleted) `build:module` script:
`cross-env BABEL_ENV=module npm run build:src -- -d module`

deleted since this was added a few years before package exports so it doesn't work well with newer node, logged [W-14321367](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001chI25YAE/view) to re-enable node ESM builds once Shane finishes his dual-publish work.